templates/zerver/help/change-your-organization-settings.md

```diff
@@ -2,8 +2,11 @@
 
 {!admin-only.md!}
 
-{!go-to-the.md!} [Organization settings](/#organization/organization-settings)
-{!admin.md!}
+1. Click on the chevron next to the gear icon in the top-right of the screen to
+reveal a dropdown menu.
+
+2. Select [Organization settings](/#organization/organization-settings) from the
+menu.
```

Comment:
I think this change is wrong; It's called "Manage organization" in the gear menu if you're an administrator...